### DISCUSSION TITLE
Exonérations ZFU

### DISCUSSION THREAD

**Discussion by mquandalle** [betagouv/mon-entreprise#2657] [Idées]
Demande utilisateur pour intégrer les exonérations pour l'embauche d'un salarié venant d'une zone franche urbaine
https://www.urssaf.fr/portail/home/employeur/beneficier-dune-exoneration/exonerations-ou-aides-a-caracter/les-zones-franches-urbaines-zfu.html

(Ça me semble moins avantageux pour l'employeur que le dispositif d'emploi franc qui concerne les même territoire me semble-t-il.)

Connexe : zones revitalisations rurales #190

**Comment by johangirod** [comment]
Un utilisateur médecin souhaiterait que les revenus ZFU qui sont exonérés d'impôts soient pris en compte dans le simulateur médecin (à investiguer).
